src/registrar/models/utility/admin_sort_fields.py

```diff
@@ -1,27 +0,0 @@
-from registrar.models.utility.admin_form_order_helper import (
-    AdminFormOrderHelper,
-    SortingDict,
-)
-
-
-class AdminSortFields:
-    # Used to keep track of how we want to order_by certain FKs
-    foreignkey_orderby_dict: list[SortingDict] = [
-        # foreign_key - order_by
-        # Handles fields that are sorted by 'first_name / last_name
-        SortingDict(
-            ["submitter", "authorizing_official", "investigator", "creator", "user"],
-            ["first_name", "last_name"],
-        ),
-        # Handles fields that are sorted by 'name'
-        SortingDict(["domain", "requested_domain"], ["name"]),
-        SortingDict(["domain_application"], ["requested_domain__name"]),
-    ]
```

Comment:
It was to reduce code complexity.  And also needed to inherit/extend formfield_for_manytomany and formfield_for_foreignkey from BaseModelAdmin, rather than replace those methods.  And also needed to change sorting to be based off of a sort order (which was in some cases a function rather than an array of fields).  So, for these reasons, needed to refactor in some way.  Decided to keep the refactor in admin.py so it was easier to find.